tests/ios/unit-test/test-imports-app/Header.h

```diff
@@ -1 +1,5 @@
-#import <TestImports-App/Header2.h>
+// Note: rules_ios currently supports both of the names.
+// - Xcode target name
+// - "Module" name
+#import <TestImports-App/Header2.h> // Path from headermap
+#import <TestImports_App/Header2.h> // Actual path
```

Comment:
CI is saying this was not found? Is `<>` the expected usage here for the actual path?

Reply:
Yeah I think that one is probably a fringe behavior that isn't 100% supported.